load-tests/Cargo.toml

```diff
@@ -8,3 +8,13 @@ goose = "^0.17"
 goose-eggs = "0.5.1"
 tokio = "^1.12"
 reqwest = "^0.11"
+jsonwebtoken = "8.3.0"
+near-primitives = "0.17.0"
+serde = "1.0.130"
+serde_json = "1.0.68"
+chrono = "0.4.19"
+near-crypto = "0.17.0"
+mpc-recovery = { path = "../mpc-recovery" }
+rand = "0.8.4"
+log = "0.4"
```

Comment:
Not a huge deal, but we don't use `log` in other crates, we use `tracing` instead

Reply:
Good point. Done.